pkg/minishift/registration/redhat.go

```diff
@@ -51,12 +62,28 @@ func (registrator *RedHatRegistrator) CompatibleWithDistribution(osReleaseInfo *
 	}
 }
 
+func (registrator *RedHatRegistrator) SSHCommand(cmd string) (string, error) {
+	startTime := time.Now()
+	out, err := registrator.SSHCommander.SSHCommand(cmd)
+	elapsed := time.Since(startTime)
+
+	if glog.V(2) {
+		if spinnerView != nil {
```

Comment:
> We don't care showing spinning wheel char / in debug logs.

that's my preferred option for now

Otherwise we might also get into a situation where we for example want to say the whole Register method (spanning multiple SSH commands). In this case the spinner gets unintentionally stopped.